crates/graphql/src/schema/objects/stats_global.rs

```diff
@@ -0,0 +1,10 @@
+use super::prelude::*;
+
+#[derive(Debug, Clone, GraphQLObject)]
+pub struct GlobalStats {
+    #[graphql(description = "Approximate number of NFTs being minted per second.")]
+    pub mint_rate: Option<f64>,
+
+    #[graphql(description = "Approximate number of NFTs being bought from listings per second.")]
+    pub purchase_rate: Option<f64>,
+}
```

Comment:
```suggestion
#[derive(Debug, Clone)]
pub struct GlobalStat {
    #[graphql(description = "...")]
    pub start_date: DateTime,

    #[graphql(description = "...")]
    pub end_date: DateTime,
}
#[graphql_object(Context = AppContext)]
impl GlobalStat {
  fn average_mint_rate(&self, ctx: &AppContext): FieldResult<i32> {}
  fn average_purchase_rate(&self, ctx: &AppContext): FieldResult<i32> {}
}
```

When switching to impl for the object the graphql macro moved to the impl and switches to `#[graphql_object(Context = AppContext)]`